hotham/src/systems/debug.rs

```diff
@@ -24,4 +24,10 @@ pub fn debug_system(engine: &mut Engine) {
         params.x = ((params.x + 0.1) % 5.) as f32;
         println!("[HOTHAM_DEBUG] params.x is now {}", params.x);
     }
+
+    if input_context.right.a_button_just_pressed() {
+        let params = &mut render_context.scene_data.params;
+        params.x = ((params.x - 0.1) % 5.) as f32;
```

Comment:
This can turn negative. I think you want to something like this: 
```suggestion
        params.x = ((params.x + 5. - 0.1) % 5.) as f32;
```